### ISSUE TITLE
Sailfish website is down

### ISSUE THREAD

**Issue by ssokolow** [rust-sailfish/sailfish#74]
All I get is `Not Found - Request ID: 01FWD26WY2GVBF6SFW1ETY4CXC` when I go to the Sailfish website.

...on a related note, have you considered hosting it using GitHub Pages?

**Comment by Kogia-sima** [comment]
We consider migrating to GitHub Pages. This could be done by using Python container and running `scripts/build-docs.py`.

**Comment by vthg2themax** [comment]
@ssokolow  What's the URL you are using? I know there's been some work on this.

**Comment by ssokolow** [comment]
I was using https://sailfish.netlify.app/ and pages below that, as was linked from https://github.com/rust-sailfish/sailfish at the time and is still linked from https://crates.io/crates/sailfish and https://lib.rs/crates/sailfish

**Comment by Anders429** [comment]
I would add that the same URL is also still linked from https://docs.rs/sailfish/latest/sailfish/.

**Comment by vthg2themax** [comment]
Wow! Thank you both for your helpful input! I will work on cleaning those up.

**Comment by Anders429** [comment]
I can confirm that the documentation link I mentioned above is correct. The link in the README (which I believe is the one @ssokolow was referring to in the links they mentioned) seems to be fine too.

**Comment by ssokolow** [comment]
The "User Guide" link just under the badges in the README still points to https://rust-sailfish.netlify.app/en/ on https://lib.rs/crates/sailfish and https://crates.io/crates/sailfish

Also, I don't know why, but the "GutHub (rust-sailfish)" tab at https://lib.rs/crates/sailfish is pointing at https://github.com/rust-sailfish/sailfish/tree/HEAD/sailfish rather than https://github.com/rust-sailfish/sailfish

**Comment by Anders429** [comment]
I was assuming that the rust-sailfish.netlify.app url was fine because the website appears to be back online.

**Comment by ssokolow** [comment]
Based on past experience, keeping it up has a worse [bus factor](https://en.wikipedia.org/wiki/Bus_factor) than using GitHub Pages and it doesn't make sense to have some links pointing to https://rust-sailfish.github.io/sailfish/ and some pointing netlify.

**Comment by vthg2themax** [comment]
@Anders429 Thank you for your comment! I'm still figuring this out.
@ssokolow That's a very good point! Sticking to GitHub pages only would be probably a good path forward. Any concerns, or comments, @Kogia-sima @jdrouet ?

**Comment by Anders429** [comment]
If you decide to go with GitHub pages, I suggest setting up a redirect from the old website to accommodate old links that may exist around the web. At least temporarily.

**Comment by vthg2themax** [comment]
@Anders429 @ssokolow @jdrouet Was there any pages left that have this problem? I think I got them all thanks to some help from the community! :+1:

**Comment by vthg2themax** [comment]
Also, I was thinking of wanting to change the comment type to be the same type as  because CTRL+K CTRL+C comments the .stpl file from VSCode. What does everyone think? @ssokolow @jdrouet @Kogia-sima @Anders429

**Comment by ssokolow** [comment]
> Was there any pages left that have this problem? I think I got them all thanks to some help from the community!

The "User Guide" link at the top of the README is still pointing at https://rust-sailfish.netlify.app/en/

> Also, I was thinking of wanting to change the comment type to be the same type as because CTRL+K CTRL+C comments the .stpl file from VSCode. What does everyone think?

Could you rephrase that? I'm not sure what you mean.

**Comment by vthg2themax** [comment]
@ssokolow Please see the updated comment, I meant follow the HTML commenting style. I guess I accidentally commented that part out. :laughing:

**Comment by Svenskunganka** [comment]
> Also, I was thinking of wanting to change the comment type to be the same type as `<!-- _Comment_goes_here -->` because CTRL+K CTRL+C comments the .stpl file from VSCode. What does everyone think?

How would that work if e.g users want to output an actual HTML comment? Sailfish comments serves as a way to comment the template, while HTML comments serves as a way to comment the rendered markup imo.

Maybe it's possible to update the sailfish vscode plugin to insert a sailfish comment rather than an HTML comment when that action is executed?

**Comment by vthg2themax** [comment]
@Svenskunganka  @ssokolow If even one user would be negatively impacted, I shall not make the change then. Thank you for your feedback! I will leave things as they are as regarding to comments in Sailfish.

On a separate note: I think I will close this in a week if there are no other spots that have bad references to the site. Please let me know if you find anything else. @Anders429 @jdrouet

**Comment by vthg2themax** [comment]
Looks like there are no additional comments. Closing now. Thanks!
